media/util.ts

```diff
@@ -192,4 +179,77 @@ export function hexQueryToArray(query: string): string[] {
     }
     if (currentCharacterSequence.length > 0 ) return [];
     return queryArray;
-}
\ No newline at end of file
+}
+
+/**
+ * @description Given two sorted collections of numbers, returns the union
+ * between them (OR).
+ */
+export function disjunction(one: number[], other: number[]): number[] {
```

Comment:
Missing the params and returns for the js-doc. I don't think we need these though, see my comment about rendering the entire selection in the `selectHandler.ts` file.